assets/src/scripts/components/well-log/index.js

```diff
@@ -29,29 +29,50 @@ const updateVisibleIds = function (store, elem, siteKey) {
 export default function (store, node, options) {
     const siteKey = getSiteKey(options.agencyCode, options.siteId);
     select(node)
-        // Initialize visible IDs from the DOM
-        .call((elem) => updateVisibleIds(store, elem, siteKey))
-        // Toggle "selected" class on state change.
-        .call(link(store, (elem, selectedId) => {
-            elem.select('.selected')
-                .classed('selected', false);
-            elem.select(`[data-local-id="${selectedId}"]`)
-                .classed('selected', true);
-        }, getSelectedConstructionId(siteKey)))
-        // Visible construction IDs on checkbox click
-        .call((elem) => {
-            elem.selectAll('input')
-                .on('click', function () {
-                    updateVisibleIds(store, elem, siteKey);
+        .select('.constructionTable')
```

Comment:
Minor style issue, to use dashes instead of camelCase for class and id names.

```suggestion
         .select('.construction-table')
```

Reply:
I try to remember this for next time. Fixed.